src/components/Layout/LayoutSidebar.tsx

```diff
@@ -172,4 +155,9 @@ export const LayoutSidebarComponent = (props: LayoutComponentUIType) => {
   );
 };
 
-export const LayoutSidebar = memo(LayoutSidebarComponent);
+export const MemoizedLayoutSidebar = memo(LayoutSidebarComponent);
+
+export const LayoutSidebar = withStyles(MemoizedLayoutSidebar, {
+  ssrStyles: () => import('components/Layout/styles.module.scss'),
```

Comment:
yes, the user can import only a specific component of the Layout or the whole Layout  ( used in most cases )
I don't think it would make sense to split the files for each component since that would create an unnecessary complexity
https://github.com/multiversx/mx-sdk-dapp-sc-explorer/tree/rm-ssr-support/src/components/Layout